src/lib.rs

```diff
@@ -153,11 +153,8 @@ pub use crate::primitives::iter::{ByteIterExt, Fe32IterExt};
 pub use crate::primitives::{Bech32, Bech32m, NoChecksum};
 
 mod error;
-/// Re-exports the hrp types from [`primitives::hrp`] to make importing ergonomic for the top level APIs.
 pub mod hrp;
```

Comment:
In 6e2be2760b5267682e0d7f3e5d5f119ee5a0783e:

Doesn't this cause compiler warnings?v

Reply:
No because the docs for this come from the `\\!` on the actual module, neat huh.

Reply:
That can't possibly be correct. `///` docs are attached to the item following them and `//!` docs are attached to the module they appear in.

Reply:
Ohh, these are on `pub mod` lines rather than `pub export crate` lines. I think that is why clippy does not complain about them.

But if they appear concatenated to `//!` comments that's just a bug.

Reply:
Bug or feature, depends on ones perspective I suppose. I found it surprising also but I'm not going to loose any sleep over it :)